articles/explainable-ai/index.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/explainable-ai/
+title: Explainable AI - An Introduction
+description: This article will go over
+author: collins-ayuya
+date: 2020-10-20T00:00:00-11:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/explainable-ai/hero.jpg
+    alt: Explainable AI example image
+---
+AI-powered systems have a lot of influence on our daily lives. A number of these systems are so sophisticated that little to no human intervention is required in their design and deployment. These systems make a lot of decisions for us every single day. They make a lot of crucial decisions that affect human lives (for example, in medicine, finance, law, defense). But how they do is often not explained.
```

Comment:
'They make a lot of crucial decisions that affect human lives' -> 'They make many decisions that affect human lives'
'But how they do is often not explained.' -> 'But how they do so is often not explained.'